tests/foreman/ui/test_role.py

```diff
@@ -448,6 +448,10 @@ def test_positive_create_with_21_filters(session):
             session.filter.create(
                 role_name, {'resource_type': resource_type, 'permission.assigned': [permission]}
             )
+        # setting the page size to 25 as 20 is default
+        view = session.filter.navigate_to(session.filter, 'All', role_name=role_name)
+        per_page_value = view.pagination.per_page
```

Comment:
You can make this call directly instead of creating a new variable. `view.pagination.per_page.fill(25)`